jit.c

```diff
@@ -896,7 +896,7 @@ struct block_map_t *block_map_alloc(uint32_t bits)
     struct block_map_t *map = malloc(sizeof(struct block_map_t));
     const uint32_t capacity = (1 << bits);
     void *ptr = malloc(capacity * sizeof(struct block_t *));
-    memset(ptr, 0, capacity);
+    memset(ptr, 0, capacity * sizeof(struct block_t *));
```

Comment:
Always use `calloc` instead of `malloc` + `memset`.

In most cases, they will be the same. In some cases, calloc() will do less work because it can skip memset() entirely. In other cases, calloc() can even cheat and not allocate any memory on modern operating systems.  However, the use of malloc() + memset() will always do the full amount of work.